test/integration/skip_helpers_test.go

```diff
@@ -0,0 +1,43 @@
+//go:build integration_tests
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/kong/kubernetes-testing-framework/pkg/clusters/types/kind"
+)
+
+type routerFlavor string
+
+const (
+	traditional           routerFlavor = "traditional"
+	traditionalCompatible routerFlavor = "traditional_compatible"
+	expressions           routerFlavor = "expressions"
+)
+
+func skipTestForRouterFlavor(t *testing.T, flavor ...routerFlavor) {
```

Comment:
Since one can pass multiple values here I'd suggest to rename this

```suggestion
func skipTestForRouterFlavors(t *testing.T, flavor ...routerFlavor) {
```